apollo-gradle-plugin/src/main/java/com/apollographql/android/gradle/ApolloCodeGenInstallTask.java

```diff
@@ -19,7 +19,7 @@
 public class ApolloCodeGenInstallTask extends NpmTask {
   static final String NAME = "installApolloCodegen";
   private static final String INSTALL_DIR = "node_modules/apollo-codegen";
-  private static final String APOLLOCODEGEN_VERSION = "0.10.5";
+  private static final String APOLLOCODEGEN_VERSION = "0.10.8";
```

Comment:
Would it make more sense to add it to the companion object in `GraphQLCompiler` and use that instead?